sigs.yaml

```diff
@@ -2815,6 +2815,7 @@ sigs:
     contact:
       slack: sig-security-tooling
     owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/cve-feed-osv/main/OWNERS
```

Comment:
@PushkarJ can you confirm if adding it as a project under the `sig-security-tooling` subproject is accurate or do we require a new subproject to be created?

Reply:
Is this mostly for documentation purposes or does this mean some permissions like approving or lgtm will be granted to owners of that subproject?

Reply:
If it's documentation purposes it's fine to keep the sub-project as sig-security-tooling